tests/regression.py

```diff
@@ -749,6 +752,14 @@ class TestMPUpdateSlimLuaHstore(MultipolygonUpdateRunner, unittest.TestCase,
     extra_params = ['--slim', '-k']
     use_lua_tagtransform = True
 
+class TestMPUpdateSlimWithFlatNodes(MultipolygonUpdateRunner, unittest.TestCase,
+                                    MultipolygonTests):
+    extra_params = ['--slim', '-F', 'flat.nodes']
```

Comment:
You'd need to add a cleanup for that file.

Reply:
Fixed in 3947e44